uc-crux-llvm/src/UCCrux/LLVM/Overrides/Skip.hs

```diff
@@ -162,37 +369,98 @@ createSkipOverride ::
-                        liftIO
-                          ( returnValue
-                              mem
-                              ret
-                              (modCtx ^. funcTypes . funcSymbol funcSym)
-                          )
-              }
+        case makeGetters modCtx funcSymb argTys clobbers of
+          Left err -> Left err
+          Right clobbers' ->
+            Right $
+              makePolymorphicLLVMOverride $
+                basic_llvm_override $
+                  LLVMOverride
+                    { llvmOverride_declare = decl,
+                      llvmOverride_args = argTys,
+                      llvmOverride_ret = retTy,
+                      llvmOverride_def =
+                        \mvar _sym args ->
+                          do
+                            liftIO $
+                              modifyIORef usedRef (Set.insert (SkipOverrideName name))
+                            Override.modifyGlobal mvar $
+                              \mem ->
+                                liftIO $
+                                  do mem' <- clobberAll mem args clobbers'
+                                     returnValue
+                                       mem'
+                                       retTy
+                                       (modCtx ^. funcTypes . funcSymbol funcSymb)
+                    }
   where
+    decl = modCtx ^. moduleDecls . funcSymbol funcSymb
     name = declName decl
     symbolName = L.decName decl
 
+    clobberOne ::
+      MemImpl sym ->
+      Ctx.Assignment (Crucible.RegEntry sym) args ->
+      ClobberSpec m args' inTy atTy ->
+      ClobberGetter arch args inTy ->
+      IO (MemImpl sym)
+    clobberOne mem args spec getter =
+      let mts = modCtx ^. moduleTypes
+      in
+        runSetup
+          modCtx
+          mem
+          ( generate
+              sym
+              modCtx
+              (clobberAtType mts spec)
+              (SelectClobbered
+                 funcSymb
+                 (deepenPtr mts (clobberSelectorCursor (clobberSelector spec))))
+              (clobberShape spec)
+          )
+          >>=
+            \case
+              (result, value) ->
+                do
+                  assume name sym (resultAssumptions result)
+                  -- The keys are nonces, so they'll never clash, so the
+                  -- bias of the union is unimportant.
+                  modifyIORef annotationRef (Map.union (resultAnnotations result))
+                  container <- getClobberGetter getter mem args
+                  let cursor = clobberSelectorCursor (clobberSelector spec)
+                  ptr <- Mem.seekPtr modCtx sym mem (clobberType spec) container cursor
+                  let mem' = resultMem result
+                  let val = value ^. Shape.tag . to getSymValue
+                  Mem.store modCtx sym mem' (clobberAtType mts spec) ptr val
+
+
+    clobberAll ::
```

Comment:
Might this more appropriately be "applyClobbers" or something similar? It isn't the case that it would always unconditionally clobber everything (at least that is true after the conditional clobber functionality is implemented)